samples/dotnet/KernelHttpServer/KernelHttpServer.csproj

```diff
@@ -36,4 +32,11 @@
       <_Parameter1>false</_Parameter1>
     </AssemblyAttribute>
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\dotnet\src\Plugins\Plugins.Document\Plugins.Document.csproj" />
+    <ProjectReference Include="..\..\..\dotnet\src\Plugins\Plugins.MsGraph\Plugins.MsGraph.csproj" />
+    <ProjectReference Include="..\..\..\dotnet\src\Plugins\Plugins.Web\Plugins.Web.csproj" />
+    <ProjectReference Include="..\..\..\dotnet\src\SemanticKernel.MetaPackage\SemanticKernel.MetaPackage.csproj" />
+  </ItemGroup>
```

Comment:
This change will be reverted back to use published NuGet packages before being committed. These changes relay on changes that are yet to be published to NuGet.